bundlegen/src/test/java/com/citrix/microapps/bundlegen/bundles/ArchiveBuilderTest.java

```diff
@@ -64,12 +65,13 @@ private void assertContent(byte[] content) {
                 "vendor1_bundle1_0.0.1/i18n/ja.json",
                 "vendor1_bundle1_0.0.1/i18n/nl.json",
                 "vendor1_bundle1_0.0.1/i18n/zh-CN.json",
-                "vendor1_bundle1_0.0.1/metadata.json"
+                "vendor1_bundle1_0.0.1/metadata.json",
+                "vendor1_bundle1_0.0.1/file.sapp"
         );
 
         assertAll(
                 () -> assertThat(BundlesArchiver.md5Hex(content)).satisfiesAnyOf(
-                        hash -> assertEquals("7ea289396d9ae1526a74160a3d6a0ba1", hash, // UNIX
+                        hash -> assertEquals("488a6431b096ec9bacbfb4e2bc0ed8a1", hash, // UNIX
                                 "Produced zip should be always exactly same on byte level"),
                         hash -> assertEquals("2e66074ce1e973c49654770a888f0c72", hash, // WINDOWS
```

Comment:
Only unix hash hash been updated, the windows variant will probably start failing. Can you try to remove `\r` characters while loading the file, please?

Reply:
Or at least add a comment that the value is outdated to speed up the failure investigation.

Reply:
Added comment about outdated hash value on Windown platform.